### ISSUE TITLE
lookup attribute

### ISSUE THREAD

**Issue by kgv** [XAMPPRocky/fluent-templates#24]
Can't find a way to get attribute message.
Is this in the plans?

**Comment by XAMPPRocky** [comment]
Thank you for your issue! I’m not sure I understand the request. Could you expand on you mean?

**Comment by kgv** [comment]
In this example:

```
login-input = Predefined value
    .placeholder = email@example.com
    .aria-label = Login input value
    .title = Type your login email
```

we have 3 attributes: `placeholder `, `aria-label` and `title`.
How to get relevant messages for these attributes ("email@example.com", "Login input value" and "Type your login email")?

**Comment by XAMPPRocky** [comment]
Ah I understand now, thank you. I am open to adding a `get_attribute` method to the `Loader` trait, and as before I’m open to accepting a PR for it, else I’ll try to work on it later in the week.

**Comment by kgv** [comment]
What do you think of [something like this](https://github.com/kgv/fluent-templates/blob/attributes/templates/src/loader/static_loader.rs#L44) implementation?
We just split `text_id` into two parts: before dot (main id) and after (attribute id).

**Comment by XAMPPRocky** [comment]
I think that would be fine, but there would need to be documentation added about that, so people know that you can lookup attributes with it.

**Comment by XAMPPRocky** [comment]
I've now implemented this in 9634f11f0d6bf1e7bb7640807329aa8239a2d62c
